root/components/home/Facts.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * @flow strict
+ * Copyright (C) 2022 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import {faMusic, faHeadphones, faCompactDisc, faUsers}
+  from '@fortawesome/free-solid-svg-icons';
+
+const Facts = (): React.Element<'section'> => (
+  <section className="counts">
+    <div className="container">
+      <div className="row">
+        <div className="col-lg-3 col-md-6">
+          <div className="count-box">
+            <FontAwesomeIcon
+              className="icon"
+              icon={faMusic}
+              size="4x"
+            />
+            <div>
+              <span>
+                {l(`1.88 M`)}
+              </span>
+              <p>
+                {l(`Artists`)}
+              </p>
+            </div>
+          </div>
+        </div>
+
+        <div className="col-lg-3 col-md-6">
+          <div className="count-box">
+            <FontAwesomeIcon
+              className="icon"
+              icon={faCompactDisc}
+              size="4x"
+            />
+            <div>
+              <span>
+                {l(`3.00 M`)}
+              </span>
+              <p>
+                {l(`Releases`)}
+              </p>
+            </div>
```

Comment:
div inside p is not valid html, apparently - but having both be spans seems like it should work?

Reply:
Yeah, I forgot about that restriction, but `<span>` should work.  We probably only need one `<span>` around the count anyway, since the entity type styling can come from the `<p>`.

Another thought: we should consider combining the strings, and probably don't want to have to have people re-translate this every time we want to update the number.  So something like `{l('<span>{total_in_millions} M</span> releases')}` and then formatting `total_in_millions` with `Intl.NumberFormat` would make sense to me.

Reply:
Fixed!

Reply:
AFAICT, it's not fixed yet :)

Reply:
Corrected